File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/engine/TouchEventListener.java
/**
 * 
 */
package com.abubusoft.xenon.engine;

/**
 * @author Francesco Benincasa
 *
 */
public interface TouchEventListener {

	/**
	 * <p>Evento relativo al touch</p>
	 * 
	 * @param type
	 * @param x
	 * @param y
	 */
	void onTouch(TouchType type, float x, float y);
}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/engine/TouchEventListener.kt
/**
 *
 */
package com.abubusoft.xenon.engine

/**
 * @author Francesco Benincasa
 */
interface TouchEventListener {
    /**
     *
     * Evento relativo al touch
     *
     * @param type
     * @param x
     * @param y
     */
    fun onTouch(type: TouchType?, x: Float, y: Float)
}